src/electionguard/facades/Hacl_Bignum256.cpp

```diff
@@ -1,13 +1,18 @@
 #include "Hacl_Bignum256.hpp"
 
 #include "../../kremlin/Hacl_Bignum256.h"
+#include "../log.hpp"
 
 #include <electionguard/constants.h>
 
+using electionguard::Log;
 using std::vector;
 
 namespace hacl
 {
+    Bignum256::Bignum256(uint64_t *elem) { context = Hacl_Bignum256_mont_ctx_init(elem); }
```

Comment:
these facades have two sides. the static side is purely functional and holds no memory internally. the instance side uses the overloads that support montgomery multiplication internally.